sunpy/timeseries/timeseriesbase.py

```diff
@@ -428,9 +442,13 @@ def concatenate(self, others, same_source=False, **kwargs):
         # Concatenate the metadata and data.
         kwargs["sort"] = kwargs.pop("sort", False)
         meta = self.meta.concatenate([series.meta for series in others])
+        # TODO: refactor this to avoid going via DataFrames
```

Comment:
just curious - by going via data frames does this do anything in regards to leap seconds? i.e. will it error if leap seconds are in data? or does it get rid of them?

Reply:
It will error, but you couldn't even create a `TimeSeries` with leapseconds before this PR, so I don't think this is a deal breaker for this PR? I've opened https://github.com/sunpy/sunpy/issues/5873 to track testing all the methods with leapsecond data.